src/form/form-dropdown.tsx

```diff
@@ -6,12 +6,14 @@ import { ErrorMessage } from './error-message/error-message';
 
 interface Props<T> extends DropdownProps<T> {
   name: string;
+  itemHeight?: number;
```

Comment:
you could move `itemHeight` from `DropdownValueProps` to `DropdownProps` and it will be added here via extending